integrations/sklearn/tests/Neptune-Scikit-learn_upgraded_libs.py

```diff
@@ -0,0 +1,252 @@
+
+gbc = GradientBoostingClassifier(n_estimators=120, learning_rate=0.12, min_samples_split=3, min_samples_leaf=2)
+
+X, y = load_digits(return_X_y=True)
+X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.20, random_state=28743)
+
+gbc.fit(X_train, y_train)
+
+## Step 3: Create Neptune experiment and log regressor summary
+
+import neptune
+from neptunecontrib.monitoring.sklearn import log_classifier_summary
+
+neptune.init('shared/sklearn-integration',
+             api_token='ANONYMOUS')
+exp = neptune.create_experiment(name='classification-example',
+                                tags=['GradientBoostingClassifier', 'classification'])
+
+log_classifier_summary(gbc, X_train, X_test, y_train, y_test, experiment=exp)
+
+# tests
+# check logs
+correct_logs_set = {'charts_sklearn'}
+for name in ['precision', 'recall', 'fbeta_score', 'support']:
+    for i in range(10):
+        correct_logs_set.add('{}_class_{}_test_sklearn'.format(name, i))
+from_exp_logs = set(exp.get_logs().keys())
+assert correct_logs_set == from_exp_logs, '{} - incorrect logs'.format(exp)
+
+# check parameters
+assert set(exp.get_properties().keys()) == set(gbc.get_params().keys()), '{} parameters do not match'.format(exp)
+
+# check estimator
+import joblib
+import os
+import tempfile
+
+import numpy as np
+
+with tempfile.TemporaryDirectory() as d:
+    exp.download_artifact('model/estimator.skl', d)
+    full_path = os.path.join(d, 'estimator.skl')
+    model = joblib.load(full_path)
+
+assert np.array_equal(model.predict_proba(X_test), gbc.predict_proba(X_test)), '{} estimator error'.format(exp)
+
+## Step 4: Stop Neptune experiment after logging summary
+
+exp.stop()
+
+## Explore Results
+
+# Scikit-learn KMeans clustering
+
+## Step 1: Import libraries
+
+from sklearn.datasets import make_blobs
+from sklearn.cluster import KMeans
+
+## Step 2: Create KMeans object and example data
+
+km = KMeans(n_init=11, max_iter=270)
+
+X, y = make_blobs(n_samples=579, n_features=17, centers=7, random_state=28743)
+
+## Step 3: Create Neptune experiment and log KMeans clustering summary
+
+import neptune
+from neptunecontrib.monitoring.sklearn import log_kmeans_clustering_summary
+
+neptune.init('shared/sklearn-integration',
+             api_token='ANONYMOUS')
+exp = neptune.create_experiment(name='clustering-example',
+                                tags=['KMeans', 'clustering'])
+
+log_kmeans_clustering_summary(km, X, n_clusters=17, experiment=exp)
+
+# tests
+# check logs
+assert list(exp.get_logs().keys()) == ['charts_sklearn'], '{} - incorrect logs'.format(exp)
+
+# check cluster labels
+assert X.shape[0] == len(km.labels_), '{} incorrect number of cluster labels'.format(exp)
+
+# check parameters
+assert set(exp.get_properties().keys()) == set(km.get_params().keys()), '{} parameters do not match'.format(exp)
+
+## Step 4: Stop Neptune experiment after logging summary
+
+exp.stop()
+
+## Explore Results
+
+# tests
```

Comment:
I think I'd drop this test potentially. 
Remember that we are testing the example (and whether things work with newer versions and not the library itself -> this should be done in contrib

Reply:
dropped.